### ISSUE TITLE
Warn guild admins if the bot is unable to work with the chonkiest role

### ISSUE THREAD

**Issue by dudeofawesome** [Virginity-Bot/virginity-bot#68]
This could be caused by the role being positioned above the bot's role.

We should probably send a message to the admins in a designated channel. Maybe the "system" channel or "widget" channel? Fall back to the vbot channel otherwise? IDK.

- Handle in:
  - [ ] `DiscordHelperService::findOrCreateBiggestVirginRole`
  - [ ] `DiscordHelperService::createBiggestVirginRole`
  - [ ] `DiscordHelperService::assignBiggestVirginRole`
- [ ] Ensure we only send warnings once
- [ ] Unset "warning sent" indicator once the error has been remedied.

**Comment by EdgarSaldivar** [comment]
Do you basically want to create additional channels that are admin only, system and widget? I suppose that would be cleaner.

**Comment by dudeofawesome** [comment]
Nah, I don't want to spam our guilds with new channels. Thinking about spam, we also would want to rate-limit the warning message.
